python/feathub/registries/registry.py

```diff
@@ -40,18 +40,22 @@ def __init__(self, registry_type: str, config: Dict) -> None:
 
     @abstractmethod
     def build_features(
-        self, features_list: List[TableDescriptor]
+        self, features_list: List[TableDescriptor], props: Optional[Dict] = None
     ) -> List[TableDescriptor]:
         """
         For each table descriptor in the given list, resolve this descriptor by
         recursively replacing its dependent table and feature names with the
-        corresponding table descriptors and features from the cache or registry.
+        corresponding table descriptors and features from the cache or registry. Then
+        recursively configure the table descriptor and its dependent table with the
```

Comment:
This statement is probably inaccurate because the dependent table will not be configured with the global properties if it is referenced by the given table descriptor by its name (rather than a table descriptor).